mxnet/buildspec-eia.yml

```diff
@@ -0,0 +1,48 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+  region: &REGION <set-$REGION-in-environment>
+  framework: &FRAMEWORK mxnet
+  version: &VERSION 1.5.1
+  os_version: &OS_VERSION ubuntu16.04
+
+  repository_info:
+    inference_repository: &INFERENCE_REPOSITORY
+        image_type: &INFERENCE_IMAGE_TYPE inference
+        root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+        repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", eia]
+        repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+  context:
+    deep_learning_container.py:
+      source: docker/artifacts/deep_learning_container.py
+      target: deep_learning_container.py
+    inference_context: &INFERENCE_CONTEXT
+      config:
+        source: docker/artifacts/config.properties
+        target: config.properties
+      entrypoint:
+        source: docker/artifacts/mms-entrypoint.py
+        target:  mms-entrypoint.py
+
+  images:
+    BuildEIAMXNetInferencePy3DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &MXNET_CPU_INFERENCE_PY3 false
```

Comment:
If a change this line from CPU to EIA, the build will not happen. Weird and have not find root cause